mne_bids/tests/conftest.py

```diff
@@ -0,0 +1,33 @@
+import os
+import platform
+
+import pytest
+from mne.utils import run_subprocess
+
+
+# WINDOWS issues:
+# the bids-validator development version does not work properly on Windows as
+# of 2019-06-25 --> https://github.com/bids-standard/bids-validator/issues/790
```

Comment:
Actually the solution is right there in the issue: simply run the validator via `noda $executable`. This should also work on all platforms.

Reply:
I just copied this over from `test_write.py` into `conftest.py` to make it re-usable across multiple test files. Should I just remove these inline comments?

Reply:
No leave the comments there so we will find them again in the future :)